readme.md

```diff
@@ -53,7 +53,7 @@ googleRegex.test('https://google.de/search'); // -> false
 
 ```js
 globToRegex('https://???.example.com/foo/*');
-// Returns /^https:\/\/...\.example\.com\/foo\/.*$/
+// Returns /^https:\/\/.?.?.?\.example\.com\/foo\/.*$/
```

Comment:
Is this right too? `.?.?.?` matches `a` or `aa` or `aaa`, while I assume `???` matches `aaa` only.

Reply:
I tested it on chrome and indeed it matches aa or aaa or just nothing

Reply:
So `?` is the same as `*` with the exception that `?` is `up to one`? while `*` is `any number of`?

How do any of them deal with the dot? We need a few more tests on Chrome and here, for example `google.com*` matches `google.com.uk` or just `google.combo` or just `google.com/extra`? Same goest with `?` and `*` when in the middle.